Dockerfile

```diff
@@ -11,20 +11,20 @@ RUN rm -rf /var/cache/apk/*
 WORKDIR /app
 
 # NPM
-COPY --chown=node:node package.json .
-COPY --chown=node:node package-lock.json .
-RUN npm ci --legacy-peer-deps
+COPY --chown=node:node package*.json .
+RUN npm ci --legacy-peer-deps --ignore-scripts
 RUN npm cache clean --force
 
-COPY --chown=node:node . .
+COPY --chown=node:node src src
```

Comment:
We also need to copy below build dependant file please:

* `nest-cli.json`

Reply:
updated now